contracts/Controller.sol

```diff
@@ -773,6 +880,61 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         emit VaultSettled(_args.owner, _args.to, address(otoken), _args.vaultId, payout);
     }
 
+    /**
+     * @notice liquidate naked margin vault
+     * @dev can liquidate different vaults id in the same operate() call
+     * @param _args liquidation action arguments struct
+     */
+    function _liquidate(Actions.LiquidateArgs memory _args) internal notPartiallyPaused {
+        require(_checkVaultId(_args.owner, _args.vaultId), "Controller: invalid vault id");
+
+        // check if vault is undercollateralized
+        // the price is the amount of collateral asset to pay per 1 repaid debt(otoken)
+        // collateralDust is the amount of collateral that should be left in vault if not fully liquidated by liquidator
```

Comment:
collateralDust is the minimum amount of collateral that can be left in the vault when a partial liquidation occurs

Reply:
@haythem96 wanna update this comment?

Reply:
done.